### ISSUE TITLE
GoogleDrive not syncing to clients

### ISSUE THREAD

**Issue by bhcopeland** [owncloud/core#20453]
### Steps to reproduce
1. Setup Google drive as per docs
2. Check I can download/access files on webui
3. Start my client sync the GoogleDrive folder. Get errors.

### Expected behaviour
Sync my GoogleDrive

### Actual behaviour
I get error 500, webdev issues. http://pastebin.com/2mTNsSem

### Server configuration
**Operating system**:
Arch Linux

**Web server:**
Nginx

**Database:**
Mysql

**PHP version:**
Php-fpm, 5.6.15-2

**ownCloud version:** (see ownCloud admin page)
ownCloud 8.2.0 (git)

**Updated from an older ownCloud or fresh install:**
Updated

**List of activated apps:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your ownCloud installation folder
```
Enabled:
  - activity: 2.1.3
  - external: 1.2
  - files: 1.2.0
  - files_external: 0.3.0
  - files_pdfviewer: 0.7
  - files_sharing: 0.7.0
  - files_texteditor: 2.0
  - files_trashbin: 0.7.0
  - files_versions: 1.1.0
  - files_videoviewer: 0.1.3
  - firstrunwizard: 1.1
  - provisioning_api: 0.3.0
  - templateeditor: 0.1
Disabled:
  - admin_migrate
  - apptemplate
  - django_auth
  - emoji
  - encryption
  - encryption_dummy
  - files_archive
  - files_locking
  - files_odfviewer
  - files_sgfviewer
  - files_svgedit
  - fluxx_compensator
  - impress
  - imprint
  - ownpad_lite
  - pong
  - pushnotifications
  - reader
  - search
  - tattoo
  - updater
  - user_external
  - user_ldap
  - user_migrate
  - user_oauth
  - user_openid_provider
  - user_persona
  - user_saml
  - user_vd
  - user_webfinger

**The content of config/config.php:**

**Are you using external storage, if yes which one:** local/smb/sftp/...
googledrive

**Are you using encryption:** yes/no
no

**Comment by Xenopathic** [comment]
The logs are just reporting that your GDrive storage is marked as failed. Please recheck in the web UI to ensure it is still working, you can force a refresh in the external storage configuration GUI by clicking on the red square next to the storage

**Comment by bhcopeland** [comment]
The WEBUI, seems to work fine. If I add a file to GoogleDrive, it appears in my Drive. If I add a file in the webui, it appears in Google Drive.

I have the sync disabled'ed on my client as it just errors out. If I start sync with my client, I get "remote.php/webdav/GoogleDrive - server replied: Internal Server Error"

**Comment by bhcopeland** [comment]
Okay.. I deleted my old GoogleDrive folder, however its got stuck on the webui, and now I cannot remove it.

I have added GoogleDrive again, under a different name. I am able to view, download, and upload all my files on the webui okay.

My client however does not sync it:

Error downloading https://url.com/owncloud/remote.php/webdav/filename - server replied: Service Unavailable

Now PROPFIND has decided to go through all my files. Which is doing about 5 requests a second, which is very slow. Not sure if there is anyway to speed this up as its now going to take ages.
